Automattic/newspack-electionkit
# Styles: First pass at updating styles

This PR is a first pass at updating the Election Kit styles. 

I mostly:
* Switched the palette to be more neutral (just to fit into the different site's palettes)
* Switched the candidates to appear in two columns where there's space 
* Adjusted the markup to pick up theme styles easier, and for styling purposes
* Added icons for the Twitter and Facebook links

I set up an [example site](https://vertical-mantis.jurassic.ninja/2020/10/01/election-kit-demo) if anyone wants to review without a test site (please ping me if you'd like login information to change the settings!).

## Testing steps

1. Apply the PR. 
2. Add the Election Kit shortcode to a page or post, and turn off AMP for that post/page; click publish.
3. Run a search ("test" returns a good array of information). 
4. Try cycling through a couple of the Newspack child themes, and confirm that things look as expected (I'd recommend including Newspack Nelson, since it has a slightly narrower content area). 
5. Try changing the custom fonts (Customizer > Typography) and colours (Customizer > Colors) and confirm that the plugin output picks up those styles (the colours will be pretty sparing, turning up on links and the search button).

## Screenshots

The information should look roughly like this:

Screenshot 1:
![image](https://user-images.githubusercontent.com/177561/94756770-4894ab80-034d-11eb-93bf-bb0fe2ee7d3e.png)

Screenshot 2:
![image](https://user-images.githubusercontent.com/177561/94756473-79281580-034c-11eb-9e2d-d1132ae2ecbc.png)

Please let me know if anything doesn't look quite right, or if you have any suggestions for changes. Also let me know if any important items don't come across that way -- I'm not American and might be missing some nuance with the information 🙂
